printer_mods/README.md

```diff
@@ -115,6 +114,7 @@ like so:
 | lakai | [Bruag Skirts](./lakai/Bruag_Skirts) | V2 350mm skirts inspired by Bruag Facades | :x: | :x: | :heavy_check_mark: | :x: |
 |leandromarceddu|[PowerSkirt](./leandromarceddu/PowerSkirt) |Modified skirtpiece for a digital power monitor|:x:|:grey_question:|:heavy_check_mark:|:grey_question:|
 |littleferris |[V0 c14 Skirt Inlet](./littleferris/V0_Skirt_Inlet) |V0 C14 Skirt inlet |:heavy_check_mark: |:x: |:x: |:x:|
+|LoCoCNC| [MGN7H Shims](./LoCoCNC/) | Shims for making bearing cartridges coplanar |:heavy_check_mark:|:x:|:x:|:x:|
```

Comment:
Update this link to point to your mod in the new directory

    [MGN7H Shims](./LoCoCNC/MGN7H_Shims)